test/new-e2e/tests/agent-subcommands/status/status_common_test.go

```diff
@@ -180,9 +181,12 @@ func (v *baseStatusSuite) TestDefaultInstallStatus() {
 		},
 	}
 
-	status := v.Env().Agent.Client.Status()
+	// the test will not run until the core-agent is running, but it can run before the process-agent or trace-agent are running
+	require.EventuallyWithT(v.T(), func(t *assert.CollectT) {
+		status := v.Env().Agent.Client.Status()
 
-	for _, section := range expectedSections {
-		verifySectionContent(v.T(), status.Content, section)
-	}
+		for _, section := range expectedSections {
+			verifySectionContent(t, status.Content, section)
+		}
+	}, 2*time.Minute, 20*time.Second)
```

Comment:
How do you compute the time here, it is by experience or there is someway to compute it ?

Reply:
It's just a (somewhat) random value 😄 
We don't have any guarantees on how long each agent takes to boot, and checking that it not the goal of this test, so I set something super high (2 minutes) just to be safe

Reply:
okay thanks